feat(issue_templates): Add issue templates for bug, features, and game data (#52)
Prophecy-eip/Backend: .github/ISSUE_TEMPLATE/bug.md

---
name: BUG
about: Bug report
title: "[BUG] Description"
labels: bug
assignees:

---

> **Before reporting a bug, please check that someone did not report the same bug as yours**

Author: ```suggestion
**Before reporting a bug, please check that someone did not report a similar one**
```
